OpenRA.Mods.Common/Lint/LintBuildablePrerequisites.cs

```diff
@@ -33,14 +33,28 @@ void Run(Action<string> emitError, Ruleset rules)
 			var providedPrereqs = rules.Actors.SelectMany(a => a.Value.TraitInfos<ITechTreePrerequisiteInfo>().SelectMany(p => p.Prerequisites(a.Value)));
 
 			// TODO: this check is case insensitive while the real check in-game is not
-			foreach (var i in rules.Actors)
+			foreach (var actorInfo in rules.Actors)
 			{
-				var bi = i.Value.TraitInfoOrDefault<BuildableInfo>();
-				if (bi != null)
-					foreach (var prereq in bi.Prerequisites)
-						if (!prereq.StartsWith("~disabled"))
-							if (!providedPrereqs.Contains(prereq.Replace("!", "").Replace("~", "")))
-								emitError($"Buildable actor {i.Key} has prereq {prereq} not provided by anything.");
+				// Catch TypeDictionary errors
+				try
+				{
+					var bi = actorInfo.Value.TraitInfoOrDefault<BuildableInfo>();
+					if (bi != null)
+					{
+						foreach (var prereq in bi.Prerequisites)
+						{
+							if (!prereq.StartsWith("~disabled"))
+							{
+								if (!providedPrereqs.Contains(prereq.Replace("!", "").Replace("~", "")))
+									emitError($"Buildable actor {actorInfo.Key} has prereq {prereq} not provided by anything.");
+							}
+						}
+					}
```

Comment:
Style nit: Remove the braces around single-line statements.

Reply:
where

Reply:
Where? `Catch` requires braces

Reply:
You added extra braces around `if (bi != null)` and the foreach.

Reply:
yes, because it is not a single line statement, thus I fixed it for not following our convention

Reply:
We only have `emitError($"Buildable actor {actorInfo.Key} has prereq {prereq} not provided by anything.");` inside. We do it like this everywhere else in the code base.

Reply:
I've been made aware that we do not. I was specifically asked to fix the same style issue multiple tiumes. Also I haven't seen this anywhere in the code base but here